src/components/Combobox/Static/StaticCombobox.stories.tsx

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
suggestion: if we have one line I'd prefer disable this rule only for this line not for whole file